src/function/scalar/map/map_from_entries.cpp

```diff
@@ -78,6 +78,7 @@ static void MapFromEntriesFunction(DataChunk &args, ExpressionState &state, Vect
 
 	// Get the arguments vector
 	auto &input_list = args.data[0];
+	// FIXME: this is assuming the input argument is always a flat vector, where is this asserted?
```

Comment:
Maybe we just assume it? Can you add some tests where the input is e.g. a dictionary vector?

Reply:
```
   81           // FIXME: this is assuming the input argument is always a flat vector, where is this asserted?
-> 82           auto arg_data = FlatVector::GetData<list_entry_t>(input_list);
```
🙃 I'll fix it in this PR too I guess haha